src/test/scala/code/api/v3_0_0/BranchesTest.scala

```diff
@@ -250,19 +311,109 @@ class BranchesTest extends V300ServerSetup with DefaultUsers {
     Branches.branchesProvider.default.set(Branches.buildOne)
   }
 
+  override def beforeEach(): Unit = {
+    super.beforeEach()
+    Connector.connector.vend.createOrUpdateBank(bankId, "exists bank", "bank", "string", "string", "string", "string", "string", "string")
+    Connector.connector.vend.createOrUpdateBranch(deletedBranch)
+    Connector.connector.vend.createOrUpdateBranch(existsBranch1)
+    Connector.connector.vend.createOrUpdateBranch(existsBranch2)
+  }
+
+  override def afterEach(): Unit = super.afterEach()
+
   feature("getBranches -- /banks/BANK_ID/branches -- V300") {
 
     scenario("We try to get bank branches for a bank without a data license for branch information") {
 
       When("We make a request v3.0.0")
       val request300 = (v3_0Request / "banks" / BankWithoutBranches.value / "branches").GET <@(user1)
       val response300 = makeGetRequest(request300)
-      Then("We should get a 400 and correct response jons format")
+      Then("We should get a 400 and correct response json format")
       response300.code should equal(400)
       response300.body.extract[BranchesJsonV300]
       json.compactRender(response300.body \ "message").replaceAll("\"", "") should include (ErrorMessages.BranchesNotFound)
     }
-   
+
+
+    scenario("We try to get bank branches those all not deleted") {
```

Comment:
It would be good to add a Tag for endpoint we test. Please take a look at https://github.com/OpenBankProject/OBP-API/blob/develop/CONTRIBUTING.md#writing-tests

Reply:
OK, fixed.